reports/Google-2018-report.md

```diff
@@ -34,36 +41,59 @@ From there, we broke it down into a series of more discrete problems.
+The licence on our repository is CC-BY and these guidelines have already been incorporated into other open source neuroimaging projects such as [NiBetaSeries](https://nibetaseries.readthedocs.io/en/latest/contributing.html).
+We also used the [“good-first-issue” principle](https://github.com/INCF/BIDS-Starter-Kit/blob/master/CONTRIBUTING.md#where-to-start-issue-labels) to periodically leave small mistakes in our repository unfixed, such as a typo or an undefined term in the glossary.
+Then, we would send out a request and guide a new contributor through the process of using the repository. 
 
 As a result, the number of unique visitors grew from less than 5 to peaks of over 50 per day.
 
 <br>
 
 :o: **Problem** International project management and communication
 
-:heavy_check_mark: **Solution** The members and collaborators of this project were located in Canada, Britain, the Netherlands, the USA, and India. The strategy for this problem was to work publicly. This meant rather than waiting until completion to share something, we would continuously post our work in progress and incorporate feedback. Specifically, Github milestones and “closing soon” tags were helpful to highlight weekly tasks and mark them for review. 
+:heavy_check_mark: **Solution** The members of this project were located in Canada ([Elizabeth DuPre](https://github.com/emdupre)), Britain ([Kirstie Whitaker](https://github.com/KirstieJane)), the Netherlands ([Dora Hermes](https://github.com/dorahermes)), the USA ([Chris Gorgolewski](https://github.com/chrisfilo) and [Chris Holdgraf](https://github.com/choldgraf/)), as well as a collaborator from India ([Madhur Tandon](https://github.com/madhur-tandon)).
+The strategy for this problem was to work publicly and to demonstrate best practise in open source project leadership.
+This meant rather than waiting until completion to share something, we would continuously post our work in progress and incorporate feedback.
+Specifically, Github milestones and “closing soon” tags on issues were helpful to highlight weekly tasks and mark them for review. 
 
-Despite these strategies, I still felt that the turning point of the project was in June when some of our team finally met in person. Summarizing an unnecessary meeting into a message is efficient, but familiarity is really important too.
+Despite these strategies, I still felt that the turning point of the project was in June when some of our team finally met in person at the [Organization for Human Brain Mapping Hackathon](https://ohbm.github.io/hackathon2018/).
+I was a recipient of a travel award ($500) for this event and it really made a huge difference to effective I was as maintainer and content creator for BIDS Starter Kit.
```

Comment:
Typo: "huge difference to effective I was as..." --> "huge difference in my effectiveness as a..."

Reply:
thanks for catching that!